curriculumBuilder/settings.py

```diff
@@ -134,6 +134,11 @@
     (LANGUAGE_CODE_DO_TRANSLATION, _('Translate')),
 )
 
+LOCALE_PATHS = (
+    os.path.join(BASE_DIR, 'translations/'),
```

Comment:
I believe this should be `os.path.join(I18nFileWrapper.static_dir(), 'translations')`